NullGuard.Fody/ExplicitModeAnalyzer.cs

```diff
@@ -49,11 +81,16 @@ public static class ExplicitModeExtensions
     const string ItemNotNullAttributeTypeName = "ItemNotNullAttribute";
     const string CanBeNullAttributeTypeName = "CanBeNullAttribute";
     const string JetBrainsAnnotationsAssemblyName = "JetBrains.Annotations";
+    const string NullableAttributeFullName = "System.Runtime.CompilerServices.NullableAttribute";
 
     public static NullGuardMode AutoDetectMode(this ModuleDefinition moduleDefinition)
     {
-        // If we are referencing JetBrains.Annotations and using NotNull attributes, use explicit mode as default.
+        if (moduleDefinition.GetTypes().Any(typeDefinition => typeDefinition.CustomAttributes.Any(attr => attr.AttributeType.FullName == NullableAttributeFullName)))
```

Comment:
The presence of this attribute on a type or a type member is the indicator that the assembly has been compiled with NRT enabled.

I gues an alternative would be to check that the assembly itself has defined NullableAttribute or NullableContextAttribute, as the compiler always seems to add those. eg.

![image](https://user-images.githubusercontent.com/384747/75621031-8ecf3400-5bdf-11ea-9189-b8b71fc8710c.png)

Reply:
> The presence of this attribute on a type or a type member is the indicator that the assembly has been compiled with NRT enabled.

I believe you're talking about `NullableContextAttribute`, but the code checks for `NullableAttribute`.

Even the presence of `NullableContextAttribute` is not [necessarily guaranteed](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA+ABATARgLABQhAxAHYCuANpQIbCUwAEMpdDhhGAzI9owMKMA3oUZjePDCkYBBABQYcABkYAXGAGdVASmGjxAXw4FxE3tIBCC5QH41mnXpOHCRgkA===), but I suppose the probability of it being present on a type is quite high.

> I gues an alternative would be to check that the assembly itself has defined NullableAttribute or NullableContextAttribute, as the compiler always seems to add those.

This would work today, but these attributes could end up being defined in the framework in the future. I'd test for `NullableContextAttribute` instead.

Reply:
Changed to `NullableContextAttribute`